apps/src/templates/studioHomepages/SpecialAnnouncement.jsx

```diff
@@ -21,25 +21,20 @@ export default class SpecialAnnouncement extends Component {
     const {isEnglish, isTeacher} = this.props;
     const headingText = isTeacher
       ? i18n.teacherAnnouncementSpecialWinter2021Heading()
-      : i18n.studentAnnouncementHoc2023Heading();
+      : i18n.announcementHoc2023DanceAIHeading();
     const descriptionText = isTeacher
       ? i18n.teacherAnnouncementSpecialWinter2021Body()
-      : i18n.studentAnnouncementHoc2023Body();
+      : i18n.announcementHoc2023DanceAIBody();
     const buttonId = isTeacher
       ? 'teacher_homepage_announcement_special_winter2021'
       : 'student_homepage_announcement_special2020';
-    const url =
-      isTeacher && isEnglish ? pegasus('/hourofcode') : pegasus('/hourofcode');
+    const url = isTeacher && isEnglish ? pegasus('/dance') : pegasus('/dance');
     const buttonText =
       isTeacher && isEnglish ? i18n.joinUs() : i18n.learnMore();
     const imageUrl =
       isTeacher && isEnglish
-        ? pegasus(
-            '/shared/images/fill-540x300/announcement/announcement_hoc2020_ai.png'
-          )
-        : pegasus(
-            '/shared/images/fill-540x300/social-media/hoc2023_social.png'
-          );
+        ? pegasus('/images/dance-hoc/dance-party-activity-ai-edition.png')
+        : pegasus('/images/dance-hoc/dance-party-activity-ai-edition.png');
```

Comment:
Am I tired or are this the same? Is there a reason not to simplify this?

Reply:
They are the same — I can definitely simplify it in this PR, but [this comment](https://github.com/code-dot-org/code-dot-org/pull/55099#discussion_r1401214069) gives more context why I didn't (tl:dr is that this component should probs be deprecated so I didn't want to mess anything up trying to refactor it)

Reply:
ah got it -- didn't connect that comment to this

Reply:
do we plan to revert this change after HoC? If so, I'm fine leaving it like this, but if this is the indefinite state, then I might ask for at least a comment

Reply:
Nope, it'll stay this way after HoC. I'll clean it up and leave a comment 🙌

Reply:
thank you! sorry for the nitpicking -- I just know I'd have the exact same question if I worked on this file in six months 😅

Reply:
Haha no worries at all, I appreciate the thorough review! I just updated it to no longer use logic to determine what to show for teachers or english — can you take another look? Updates are in this commit: https://github.com/code-dot-org/code-dot-org/pull/55099/commits/fc71173dcf330170f0d095db8ae56bb0a1fdd8bf